client/src/components/ui/Dropdown/Dropdown.style.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+interface SVG {
+    imageSize?: {
+      width: string;
+      height: string;
+    };
+}
+
+const SVG = styled.img<SVG>`
```

Comment:
```suggestion
const SVGElement = styled.img<SVG>`
```
because you are using interface with the same name above, it could be misleading